setup.py

```diff
@@ -200,6 +199,7 @@ def run_tests(self):
         'stsci.image>=2.3',
         'stsci.imagestats>=1.4',
         'stsci.stimage>=0.2',
+        'tweakwcs>=0.5.1',
```

Comment:
`stsci.stimage` can be removed as a direct dependency as well.  It is no longer a dependency of `jwst`, but is now a dependency of `tweakwcs`.